README.md

```diff
@@ -1,148 +1,431 @@
+The EULA can be found in `/opt/mscs/worlds/myWorld` where `myWorld` 
+is the name given to the world you created. 
 
-    #By changing the setting below to TRUE you are indicating your agreement to our EULA (https://account.mojang.com/documents/minecraft_eula).
-    eula=true
+After accepting the EULA simply start the server using the same command above, 
+and you're all set!
 
-### Requirements
-We've made an attempt to utilize only features that are normally installed in
-most Linux and UNIX environments in this script, but there are a few
-requirements that this script has that may not already be in place:
-* Java JRE     - The Minecraft server software requires this.
-* Perl         - Most, if not all, Unix and Linux like systems have this
-                 preinstalled.
-* Python       - Required by the Minecraft Overviewer mapping software.
-* GNU Wget     - Allows the script to download software updates via the
-                 internet.
-* rdiff-backup - Allows the script to efficiently run backups.
-* Socat        - Allows the script to communicate with the Minecraft server.
-* Iptables     - Although not explicitly required, a good firewall should be
-                 installed.
+### Importing an existing world
+### Import Existing Worlds
 
-If you are running Debian or Ubuntu, you can make sure that these are
-installed by running:
+You just need to create a new directory in the worlds folder for the world you wish to import.
+Suppose the world you wish to import is called `alpha`, you would create a new folder in
+`/opt/mscs/worlds` with the same name as the world, then copy the data files over to that new directory.
 
-    sudo apt-get install default-jre perl python wget rdiff-backup socat iptables
+If the directory containing the world `alpha` you wish to import looks like this:
 
-### Mapping Software
-The script uses the [Minecraft Overviewer](http://overviewer.org) mapping
-software to generate maps of your worlds.  Minecraft Overviewer is a
-command-line tool for rendering high-resolution maps of Minecraft worlds. It
-generates a set of static html and image files and uses the Google Maps API to
-display a nice interactive map.
+    $ ls
+    alpha
+    banned-ips.txt
+    banned-players.txt
+    crash-reports
+    logs
+    ops.txt
+    server.properties
+    white-list.txt
 
-You can [download](http://overviewer.org/downloads) premade binaries for
-supported systems, or build your own binary from source if needed.
+You can just copy your world into the worlds directory:
 
-Repositories for automatic installation are also available:
-* [Debian/Ubuntu](http://overviewer.org/debian/info)
-* [RHEL/CentOS/Fedora](http://overviewer.org/rpms/info)
+    mkdir /opt/mscs/worlds/alpha  // Makes a directory called `alpha` in the `/worlds/` folder
+    cp -R * /opt/mscs/worlds/alpha // Copies the `alpha` world data folder into `/opt/mscs/worlds/alpha`
+    
```

Comment:
I don't like having the comments in line on commands that people are going to want to copy-paste.  As is, if somone were to copy-paste your mkdir command, they would end up with directories called `Makes`, `a`, `directory`, etc in whatever directory they happen to be in.

 Also, I think it would be best to mention running `mscs create alpha` to setup the `server.properties` and `mscs.properties` files at this point